weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/PIL.Image.Image/PILImageImage.tsx

```diff
@@ -21,10 +23,17 @@ export const PILImageImage: React.FC<{
   data: PILImageImageTypePayload;
 }> = props => {
   const {useFileContent} = useWFHooks();
+
+  const imageFileName = Object.keys(props.data.files)[0] as
```

Comment:
This feels overly optimistic. I would instead get the keys and look for exact matches, then have a failure case